src/Analysis/GWASApp/Steps/ConfigureGWAS/ShowFinalPopulationSizeMessages.jsx

```diff
@@ -0,0 +1,27 @@
+import ACTIONS from '../../Utils/StateManagement/Actions';
+import { minimumRecommendedCohortSize, MESSAGES } from '../../Utils/constants';
+
+const ShowFinalPopulationSizeMessages = (dispatch, finalPopulationSizes) => {
+  const checkFinalPopulationSizes = () => {
+    let hasSizeIssue = false;
+    finalPopulationSizes.forEach((obj) => {
+      if (obj?.size < minimumRecommendedCohortSize) {
+        hasSizeIssue = true;
+      }
+    });
+    return hasSizeIssue;
+  };
```

Comment:
maybe this function can be moved to a shared place? It is now also present in ConfigureGWAS.jsx

Reply:
as agreed, duplication will be addressed in https://ctds-planx.atlassian.net/browse/VADC-509